app/templates/packages/view.html

```diff
@@ -6,6 +6,14 @@
 	{{ package.title }}
 {% endblock %}
 
+<meta name="og:title" content="{{ package.title }}"/>
```

Comment:
this needs to be in the `<head>` element, so put it in the headextra block:

```liquid
{% block headextra %}
	<meta name="og:title" content="{{ package.title }}"/> 
	<meta name="og:description" content="{{ package.short_desc }}"/>
	<meta name="description" content="{{ package.short_desc }}"/>
	<meta name="og:url" content="{{ package.getDetailsURL() }}"/>
	{% if package.getMainScreenshotURL() %}
		<meta name="og:image" content="{{ package.getMainScreenshotURL() }}"/>
	{% endif %}
{% endblock %}
```

Reply:
Done.